app/logic/Deployments.scala

```diff
@@ -57,4 +64,17 @@ object Deployments {
       event.note,
       event.result.getOrElse(Succeeded)
     )
+
+  def buildLinks(deployment: Deployment, jiraResponse: Option[WSResponse]) = Kleisli[Future, JIRA.Context, Seq[Link]] {
```

Comment:
ok sure. Yeh it does look a bit weird as the updatedLinks aren't actually be used anywhere